core/server/monkey-patch.js

```diff
@@ -0,0 +1,28 @@
+var TimezonedDate = require('timezoned-date'),
+    lodash = require('lodash'),
+    clonedeep = require('lodash.clonedeep'),
+    toString = require('lodash.tostring');
+
+// ######## LODASH
+// timezoned-date does not work with lodash.cloneDeep 3.x
+lodash.cloneDeep = clonedeep;
+
+// @TODO: reason?
+lodash.toString = toString;
+
+// ######## NATIVE JS DATE
+// remember original timezone offset of server
+global.ServerTimezoneOffset = new Date().getTimezoneOffset();
+
+// timezoned-date is able to force UTC at process level
+// we only use it when local system is NOT UTC
+if (ServerTimezoneOffset !== 0) {
```

Comment:
You're right about this fact, but DST is in general a problem, we risk an hour anyway for each TZ.
There is a comment in my migration script which says "we cannot exactly recalculate the datetimes into UTC, because of DST". As long as this check notice an offset, we are OK with that to run the system in UTC.